its/ruling/src/test/resources/guava/squid-S4838.json

```diff
@@ -0,0 +1,14 @@
+{
+'com.google.guava:guava:src/com/google/common/collect/MapMakerInternalMap.java':[
+3517,
```

Comment:
This seems to be a FP

Reply:
Agreed. As discussed, I updated the rule to only raise an issue when the iteration variable is actually downcast in the loop statement.